docs/guides/options.md

```diff
@@ -1,138 +1,312 @@
+  - [`sources`](#sources)
+  - [`techOrder`](#techorder)
+- [Component Options](#component-options)
+  - [`children`](#children-1)
+  - [`${componentName}`](#componentname)
+- [Tech Options](#tech-options)
+  - [`${techName}`](#techname)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Standard `<video>` Element Options
+
+Each of these options is also available as a [standard `<video>` element attribute][video-attrs]; so, they can be defined in all three manners [outlined above](#setting-options). Typically, defaults are not listed as this is left to browser vendors.
+
+### `autoplay`
+
+> Type: `Boolean`
+
+If `true`/present as an attribute, begins playback automatically.
+
+> **Note:** As of iOS 10, Apple offers `autoplay` support in Safari. For details, refer to ["New <video> Policies for iOS"][ios-10-updates].
+
+### `controls`
+
+> Type: `Boolean`
+
+Determines whether or not the player has controls that the user can interact with. Without controls the only way to start the video playing is with the `autoplay` attribute or through the Player API.
+
+### `height`
+
+> Type: `String|Number`
+
+Sets the display height of the video player in pixels.
+
+### `loop`
+
+> Type: `Boolean`
+
+Causes the video to start over as soon as it ends.
+
+### `muted`
+
+> Type: `Boolean`
+
+Will silence any audio by default.
+
+### `poster`
+
+> Type: `String`
+
+A URL to an image that displays before the video begins playing. This is often a frame of the video or a custom title screen. As soon as the user hits "play" the image will go away.
+
+### `preload`
+
+> Type: `String`
+
+Suggests to the browser whether or not the video data should begin downloading as soon as the `<video>` element is loaded. Supported values are:
+
+- `'auto'`
+
+  Start loading the video immediately (if the browser supports it). Some mobile devices will not preload the video in order to protect their users' bandwidth/data usage. This is why the value is called 'auto' and not something more conclusive like `'true'`.
+
+  _This tends to be the most common and recommended value as it allows the browser to choose the best behavior._
+
+- `'metadata'`
+
+  Load only the meta data of the video, which includes information like the duration and dimensions of the video.
+
+- `'none'`
+
+  Don't preload any data. The browser will wait until the user hits "play" to begin downloading.
+
+### `src`
+
+> Type: `String`
+
+The source URL to a video source to embed.
+
+### `width`
+
+> Type: `String|Number`
+
+Sets the display height of the video player in pixels.
+
+## Video.js-specific Options
+
+Each option is `undefined` by default unless otherwise specified.
+
+### `aspectRatio`
+
+> Type: `String`
+
+Puts the player in [fluid](#fluid) mode and the value is used when calculating the dynamic size of the player. The value should represent a ratio - two numbers separated by a colon (e.g. `"16:9"` or `"4:3"`).
+
+### `children`
+
+> Type: `Array`
```

Comment:
This is the same as the component option and I wanted to keep all the `children` stuff together. I'll link to it, though!